maxim/tests/test_langgraph.py

```
@@ -139,47 +134,71 @@ class GraphConfig(TypedDict):
+
+        # Run the async test
+        asyncio.run(run_test())
+
+        # Flush the logger and verify logging
+        logger.flush()
+        self.mock_writer.print_logs_summary()
+
+        # LangGraph creates multiple traces due to the agent workflow
+        # We expect at least one trace to be created
+        all_logs = self.mock_writer.get_all_logs()
+        self.assertGreater(len(all_logs), 0, "Expected at least one log to be captured")
```

Ruff (0.11.9)

Use a regular `assert` instead of unittest-style `assertGreater`

Replace `assertGreater(...)` with `assert ...`

(PT009)